docs/maps/indexing-and-styling-geojson-data.asciidoc

```diff
@@ -0,0 +1,257 @@
+[role="xpack"]
```

Comment:
Is this the old file?  Is it ok to delete it?

Reply:
Deleted!